internal/util/endpoints.go

```diff
@@ -30,6 +31,12 @@ type test struct {
 	httpMethod string
 }
 
+// httpClient is the client that used for making HTTP requests to Cloud Run services. It is the same as the
+// http.DefaultClient, except for the fact that it has a timeout for requests made.
+var httpClient = &http.Client{
```

Comment:
This is probably fine here, but I'd say it's more typical to do timeouts on a per-request basis using a context. Usually you would be making multiple types of requests with a single http.Client and would want more granular control on individual timeouts. To do this:
* Create a context with a timeout using `WithTimeout`
* Use https://golang.org/pkg/net/http/#Request.WithContext to attach this to the request passed to your client.

Reply:
Being of a defensive mindset I prefer to have a sane default value for this, but no objection to the other approach. One consideration is to keep the existing implementation, but use WithTimeout in the HTTP requests to apply overrides as needed.

An interesting follow-up might be to read a per operation timeout from the configuration along the lines of this custom OpenAPI Extension: https://cloud.google.com/endpoints/docs/openapi/openapi-extensions#deadline

Reply:
Resolved, thank you both!

With regards to configurable timeouts for each test, we haven't added the option yet for users to specify their own OpenAPI file yet (we're currently only doing a default GET request to the root endpoint and expecting a 200 status code), but we'll keep Google's OpenAPI Extension in mind while implementing it.